core/controllers/feedback.py

```diff
@@ -57,13 +58,36 @@ class ThreadListHandler(base.BaseHandler):
 
     @acl_decorators.can_play_exploration
     def get(self, exploration_id):
+
+        feedback_dicts = [
```

Comment:
There is a lot of duplication in the code below, can we somehow optimize it?

Reply:
I tried to remove the duplicates by the `get_updated_thread_dict` function but am not sure where should I keep that function. Is it ok in `feedback.py`?